testrunner/execute.go

```diff
@@ -337,7 +340,7 @@ type config struct {
 func findAdditionalTestFlags(input_dir string) []string {
 	configContent, err := os.ReadFile(filepath.Join(input_dir, ".meta", "config.json"))
 	if err != nil {
-		log.Printf("failed to read config.json: %v", err)
+		log.Printf("warning: config.json could not be read: %v", err)
```

Comment:
This is not related to the issue, just a minor improvement to make it clear that this is not an error. (The case happens for a lot of the test folders.)